Verifica Verbale Esterno 2025-03-24 (#204)
SevenBitsSwe/7BitsDocs: PB/Documentazione Esterna/Verbali Esterni/VerbaleEsterno-2025-03-24.tex

%\graphicspath{{../../../images/}}

%path per quando caricare in repo
\graphicspath{{images/}}

%cambio misure della pagina
\geometry{a4paper,left=20mm,right=20mm,top=20mm}

\title{Verbale Esterno del meeting in data 2025-03-24} % Sostituire la data
\date{A.A 2024/2025}

\renewcommand*\contentsname{Indice}
\begin{document}
\maketitle
\center 
\includegraphics[width=0.25\textwidth]{LogoUnipd}\\
\includegraphics[width=0.25\textwidth]{Sevenbitslogo}\\
sevenbits.swe.unipd@gmail.com\\
\vspace{2mm}

\newpage
\raggedright
\tableofcontents

\newpage
\section{2025-03-24} % Inserire la data in formato yyyy-mm-dd
\subsection{Durata e partecipanti}
\begin{itemize}
\item Ora: 10:30 - 10:45; % Sostituire con l'orario corretto del meeting
\item Partecipanti: 	
	\begin{itemize}
            \item SevenBits:
            \begin{itemize}
                \item Giovanni Cristellon;
                \item Gusella Manuel;
                \item Peruzzi Uncas;
                \item Piva Riccardo;
                \item Pivetta Federico;
                \item Rubino Alfredo;
                \item Trolese Leonardo.
	    \end{itemize}
            \item SyncLab:
            \begin{itemize}

>                 \item Zorzi Daniele;

Reviewer: ```suggestion
                \item Dorigo Andrea;
```

                \item Pallaro Fabio.
	    \end{itemize}
	\end{itemize}
\item Piattaforma: Google meet (online)
\end{itemize}

\subsection{Ordine del giorno}
\begin{itemize}
    \item Aggiornamento sullo stato di avanzamento dell’MVP;
    \item Definizione della data di presentazione del prodotto \textit{Near You} in azienda prima della revisione PB.
\end{itemize}

\subsection{Oggetto}
Incontro organizzativo con i proponenti Andrea Dorigo, Fabio Pallaro, incentrato sull'allineamento gruppo e azienda su quanto svolto dallo scorso SAL ad oggi.

\subsection{Sintesi}

> Durante l'incontro sono state esplicate le modifiche apportate al progetto in seguito all'ultimo sprint. Inoltre, il gruppo ha deciso di fissare una data definitiva per la presentazione del prodotto finale in azienda.

Reviewer: ```suggestion
Durante l'incontro sono state esplicate le modifiche apportate al progetto a seguito dell'ultimo sprint. Inoltre, il gruppo ha deciso di fissare una data definitiva per la presentazione del prodotto finale in azienda.
```

    
    \subsubsection{MVP}

>     È stata aggiornata l'azienda sullo stato di sviluppo dell'MVP. In particolare, è stata aggiunta una tabella nell'interfaccia grafica che tiene traccia del numero di pubblicità generate per ciascuna azienda. Il gruppo ha fatto presente che, essendo l'MVP nella fase finale dello sviluppo, non ci sono altre modifiche significative degne di essere discusse, in quanto ci si sta concentrando sul completamento della redazione dei documenti necessari per la consegna del PB.

Reviewer: ```suggestion
    È stata aggiornata l'azienda sullo stato di sviluppo dell'MVP. In particolare, è stata aggiunta una tabella nell'interfaccia grafica che tiene traccia del numero di pubblicità generate per ciascuna azienda. Il gruppo ha fatto presente che, essendo l'MVP nella fase finale dello sviluppo, non ci sono altre modifiche significative degne di essere discusse, in quanto ci si sta concentrando sul completamento della redazione dei documenti necessari per la consegna finale.
```
